LICENSE_LOREMPIXEL

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Why a bash script file is called `LICENSE_LOREMPIXEL`? Rename to `fetch-images-from-lorempixel.sh`?